battle.go

```diff
@@ -187,10 +187,13 @@ func (b *Battle) SimulateRound() ([]Transaction, bool) {
 
 			// use the move
 			move := user.Moves[t.Move]
-			accuracy := move.Accuracy
+			accuracy := float64(move.Accuracy)
```

Comment:
If I do this, the defog multiplier makes accuracy 0, because 3/5 is 0 with integers. One workaround was accuracy = accuracy * 3 / 5 which I didn't like as much.

Reply:
The operations get evaluated left to right.

https://play.golang.org/p/vYjmYu9-VkX

Reply:
This is kinda weird: https://play.golang.org/p/tS08qwc4tFi